slug.opam

```diff
@@ -14,10 +14,11 @@ depends: [
   "dune" {build}
   "opam-lock" {dev}
   "yojson" {dev}
```

Comment:
This is an oversight on my part. "yojson" is need for `data_generate` to work. 
`data_generate.exe` reads json map and outputs `(string * string) list` .